packages/client/src/components/Cart/index.tsx

```diff
@@ -38,7 +43,7 @@ function Cart({cartList, handleClick, handleRemoveCart}: CartProps) {
           </TotalPrice>
         </>
         <>
-          <RemoveAllButtpn>장바구니 비우기</RemoveAllButtpn>
+          <RemoveAllButtpn onClick={() => handleRemoveAllCart()}>장바구니 비우기</RemoveAllButtpn>
```

Comment:
handl
```suggestion
          <RemoveAllButtpn onClick={handleRemoveAllCart}>장바구니 비우기</RemoveAllButtpn>
```

이렇게 사용해도 되겠네요~~